controllers/operands/kubevirt.go

```diff
@@ -303,6 +310,30 @@ func NewKubeVirt(hc *hcov1beta1.HyperConverged, opts ...string) (*kubevirtcorev1
 	return kv, nil
 }
 
+func isAnnotationStateMeetingRequirements(hc *hcov1beta1.HyperConverged, annotations map[string]string) bool {
+	_, found := annotations[kubevirtcorev1.EmulatorThreadCompleteToEvenParity]
+
+	if (hc.Spec.FeatureGates.AlignCPUs == nil || !*hc.Spec.FeatureGates.AlignCPUs) && found {
+		return false
+	} else if (hc.Spec.FeatureGates.AlignCPUs != nil && *hc.Spec.FeatureGates.AlignCPUs) && !found {
+		return false
+	}
+
+	return true
```

Comment:
I think what we really want to check here is that `(required KV's annotation exists) XOR (found KV's annotation exists)`. I mean - here I don't care if the annotation is reuired or not. This is already done in `NewKubeVirt` (by`setAnnotationsToReqState`).

Reply:
Good idea!
Done.